Todo example with js doesn't work

Trying to run the todo example but only with js results in this error,
```
Error: Cannot find module 'src/crud' imported from '~/my-rakkas-app/src/routes/todo/Todo.jsx'
    at nodeImport (~/my-rakkas-app/node_modules/vite/dist/node/chunks/dep-ca21228b.js:52451:25)
    at ssrImport (~e/my-rakkas-app/node_modules/vite/dist/node/chunks/dep-ca21228b.js:52374:20)
    at eval (/src/routes/todo/Todo.jsx:9:37)
    at async instantiateModule (~/my-rakkas-app/node_modules/vite/dist/node/chunks/dep-ca21228b.js:52420:9) {
  code: 'ERR_MODULE_NOT_FOUND'
```

I fixed it by adding this to the vite.config.js,
```
resolve: {
    alias: {
      'src': path.resolve(__dirname, 'src'),
    },
  }
```
And created jsconfig.json for autocomplete,
```
{
  "compilerOptions": {
    "paths": {
      "src/*": ["./src/*"]
    }
  }
}
```
I tried looking at the example code to create a PR but seems like it does some compilation so wanted to clarify first.

--- a/packages/create-rakkas-app/scripts/create-templates.ts
+++ b/packages/create-rakkas-app/scripts/create-templates.ts
@@ -85,7 +85,14 @@ async function main() {
 	contents = fs.readFileSync(dest + "/vite.config.js", "utf8");
 	contents = contents
 		.replace(`tsconfigPaths(), `, "")
-		.replace(/import tsconfigPaths from "vite-tsconfig-paths";\r?\n/, "");
+		.replace(
+			/import tsconfigPaths from "vite-tsconfig-paths";/,
+			`import path from "node:path";\n`,
+		)
+		.replace(
+			"plugins",
+			`resolve: { alias: { src: path.resolve(__dirname, "src") } },\nplugins`,
+		);
 	fs.writeFileSync(dest + "/vite.config.js", contents);
 
 	const ESLINT_CONFIG = `require("@rakkasjs/eslint-config-js/patch");
@@ -97,6 +104,17 @@ async function main() {
 
 	fs.writeFileSync(dest + "/.eslintrc.cjs", ESLINT_CONFIG, "utf8");
 	fs.writeFileSync(dest + "/.prettierrc", "{}", "utf8");
+
+	const JSCONFIG = `{
+			"compilerOptions": {
+				"paths": {
+					"src/*": ["./src/*"]
+				}
+			}
+		}
+	`;
+
+	fs.writeFileSync(dest + "/jsconfig.json", JSCONFIG, "utf8");
 
 	await run(`prettier ${dest}/.. --write`);
 }

